.github/workflows/cicd-main.yml

```diff
@@ -3574,17 +3574,60 @@ jobs:
         inference.outfile_path=/tmp/nlp_mcore_t5_lora_tuning_tp2/out.jsonl
 
 
+  L2_HF_Transformer_PEFT:
+    needs: [ cicd-test-container-setup ]
+    uses: ./.github/workflows/_test_template.yml
+    if: contains(fromJSON(needs.cicd-test-container-setup.outputs.test_to_run), 'L2_HF_Transformer_PEFT') || needs.cicd-test-container-setup.outputs.all == 'true'
+    with:
+      RUNNER: self-hosted-azure
+      SCRIPT: |
+        python tests/collections/llm/hf/peft.py --model /home/TestData/nlp/hf_gemma/hf_gemma_2b --max-steps 10
```

Comment:
I needed to add `--strategy ddp` to use  `torch.distributed.get_rank()` while saving ckpt.

Reply:
@HuiyingLi thank you, can you push your fix to this branch as well?

Reply:
Added --disable-ckpt option for the time being and we can remove it once the checkpoint work is done.